app/src/main/java/com/khanhhuy/movie_db/adapter/ViewPagerAdapter.java

```diff
@@ -0,0 +1,38 @@
+package com.khanhhuy.movie_db.adapter;
+
+import android.support.v4.app.Fragment;
+import android.support.v4.app.FragmentManager;
+import android.support.v4.app.FragmentStatePagerAdapter;
+
+import com.khanhhuy.movie_db.fragment.FragmentFavorite;
+import com.khanhhuy.movie_db.fragment.FragmentHome;
+
+/**
+ * Created by yeu_thuong on 10/31/2017.
+ */
+public class ViewPagerAdapter extends FragmentStatePagerAdapter {
+    private int mNumberOfTabs;
+
+    public ViewPagerAdapter(FragmentManager fragmentManager, int numberOfTabs) {
+        super(fragmentManager);
+        mNumberOfTabs = numberOfTabs;
+    }
+
+    @Override
+    public Fragment getItem(int position) {
+        switch (position) {
+            case 0:
+                FragmentHome fragmentHome = new FragmentHome();
```

Comment:
E tham khảo cách tạo instance cho activity và fragment
2.11 Argument in Activity and Fragment
https://github.com/framgia/coding-standards/blob/master/eng/android/codingstyleguide.md